.github/workflows/test.yaml

```diff
@@ -27,3 +27,6 @@ jobs:
       - name: Lint
         run: make lint
         shell: bash
+      - name: License
+        run: make gen-license
```

Comment:
I think you need to add another step to verify that the file has not be change avec this command execution